tests/backup_helper.go

```diff
@@ -6933,3 +6933,53 @@ func ValidateCustomResourceRestores(ctx context1.Context, orgID string, resource
 	}
 	return nil
 }
+
+// func InjectNetworkErrorInNodes inject error in application nondes by dropping packets or introdiucing delay in packet tramission.
+// nodes=> list of nodes where network injection should be done.
+// errorInjectionType => pass "delay" or "drop"
+// operationType => add/change/delete
+// dropPercentage => intger value from 1 to 100
+// delayInMilliseconds => 1 to 1000
+func InjectNetworkErrorInNodes(nodes []node.Node, errorInjectionType string, operationType string, dropPercentage int, delayInMilliseconds int) error {
```

Comment:
Also should we move it to common.go as other teams will also be able to use it? @mkoppal-px what do you think??

Reply:
Moved the function to Inst().N / ssh.go